release_notes/v0.8.md

```diff
@@ -0,0 +1,174 @@
+# OPEA Highlights
+- In this release, OPEA supports large vision models and provides an initial Agent microservice. It also integrates more components, such as vLLM on Ray, giving users more choices for different AI functionalities. We have expanded the application range, enabling ChatQnA on AIPC and validating it on AWS. Additionally, more examples, such as FAQGen, are coming. Deployment is important, Kubernetes manifests for ChatQnA, DocSum, CodeGen and CodeTrans are created.
```

Comment:
Can we add some highlights for GMC as well?  Like 
GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps.

Reply:
@mkbhanda @yipan ^^

Reply:
Please add, based on @irisdingbj input 
"GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice Applications on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps."